### PR TITLE
Removed user creation scripts

### PR DESCRIPTION
**Pull request by matthewdfuller**
- In v3, the AuthorizedKey is used for SSH. The private git users repo is no longer required. This PR removes the service units and scripts required to create separate users. KLAM-SSH will further this in the future.

### CODE REVIEW

File: v3/fleet/control-jenkins.service [hunk 1]
```diff
@@ -11,7 +11,7 @@ Restart=always
 RestartSec=4
 TimeoutStartSec=0
 ExecStartPre=-/usr/bin/sudo rm -rf /home/core/control-jenkins
-ExecStartPre=/usr/bin/git clone git@github.com:adobe-platform/control-jenkins /home/core/control-jenkins
+ExecStartPre=/usr/bin/git clone https://github.com/adobe-platform/control-jenkins.git /home/core/control-jenkins
```
Comment (matthewdfuller) [inline v3/fleet/control-jenkins.service:14]: Since we're getting rid of the private key, can't clone SSH here.

**Comment by matthewdfuller** [comment]
A/B works -> no more mesos-users. Everything else is good

File: v3/fleet/control-jenkins.service (hunk 1 above)
Comment (chr0n1x) [inline v3/fleet/control-jenkins.service:14]: 👍 😄

**Comment by chr0n1x** [comment]
👍

**Pull request merged**